account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -39,6 +40,14 @@ import type { RequestGasAndPaymasterAndDataRequest } from "../actions/types.js";
 import { PermitTypes, EIP712NoncesAbi } from "../gas-manager.js";
 import type { PermitMessage, PermitDomain } from "../gas-manager.js";
 
+type Context = {
+  policyId: string | string[];
+  erc20Context?: {
```

Comment:
shouldn't this be `policyToken?: PolicyToken`?

I do think erc20Context as a name is much clearer than policyToken fwiw, so maybe we should do that below